Packs/Reco/Integrations/Reco/Reco.yml

```diff
@@ -272,6 +272,34 @@ script:
       - ALERT_STATUS_CLOSED
     description: update alert status in Reco.
     name: reco-change-alert-status
+  - arguments:
+    - description: user email address.
+      name: email_address
+      required: true
+    outputs:
+    - contextPath: Reco.User.email_account
```

Comment:
It is not allowed to use underscore in the contextPath and context Keys, please change it and change the response from the API as well.